### DISCUSSION TITLE
BlueMap Integration like BlueMap-Essentials

### DISCUSSION THREAD

**Discussion by hamcoremusic** [Zrips/CMI#4646] [Feature Requests]
**Description of feature/improvement request:**
Would love to see a BlueMap API integration to show warps, homes, etc. on the map
---
### Detailed explanation of what it should do, when and in what situation: ###
There is an API to link into BlueMap here: https://github.com/BlueMap-Minecraft/BlueMap/wiki/3rd-party-support

Here is an exmaple of an essentials bridge that currently works with BlueMap: https://www.spigotmc.org/resources/bluemap-essentials.84365/

### Basic explanation as to why this feature is needed and/or useful to have: ###
I think this would be a fun thing to have since BlueMap looks like a decent plugin for server maps. It also has support for Essentials so I thought maybe CMI can take advantage of this.

This may even work for your other plugins!

**Comment by nuttapillar** [comment]
I agree. I just switched to CMI

**Comment by herobrinein** [comment]
Also
